### ISSUE TITLE
On connect, Hexchat performing: /ctcp fuck off

### ISSUE THREAD

**Issue by dickcox** [hexchat/hexchat#1517]
I have a bot with the nickname 'fuck' on efnet, and whenever someone on hexchat connects to the network it sends a /ctcp to my bot saying "off". Not sure what the purpose of this is, but it would be great if this "feature" could be removed.

Thank you!

**Comment by berrythesoftwarecodeprogrammar** [comment]
its part of the irc rfc dude. close this issue

**Comment by dickcox** [comment]
hexchat is the only client that does this..

**Comment by dickcox** [comment]
Also, I can't find anything in the below links about this..

https://tools.ietf.org/html/rfc1459
https://tools.ietf.org/html/rfc2812

**Comment by revmischa** [comment]
where did the previous comments go?

**Comment by dickcox** [comment]
Tested with some other people who use hexchat on this issue, it might be related to ctcp replies not parsing %s correctly? Is what they said.

**Comment by dickcox** [comment]
@revmischa sorry, the comments were from someone who thought this issue was a joke.

**Comment by berrythesoftwarecodeprogrammar** [comment]
yeah i opened #1518 
but we wont know until the peolpe whos clients are doing that tell us if theyve messed with ctcp reply or auto replace settings

**Comment by dickcox** [comment]
This ended up being ctcp version in version replies was "ctcp fuck off" instead of "ctcp %s fuck off". Not sure if it's a bug or whatnot, as I've never used hexchat.

**Comment by Arnavion** [comment]
You could make the case that there is no reason for a ctcp response to not be sent to the sender of the request, so HC *could* be changed to interpret ctcp without the %s as ctcp %s.

**Comment by dickcox** [comment]
Yeah, I get that. Just weird that one bot gets a lot of ctcp's from people on hexchat.

**Comment by Arnavion** [comment]
I reopened the issue for a reason.

**Comment by dickcox** [comment]
Ok cool. My bad

**Comment by mk-pmb** [comment]
I for one like how easy it is to make a CTCP query trigger a CTCP message to someone else.
